### PR TITLE
txm: clean up eth/evm prefixes

### PR DESCRIPTION
**Pull request by aalu1418**
tickets: 
* https://smartcontract-it.atlassian.net/browse/BCI-1198
* https://smartcontract-it.atlassian.net/browse/BCI-1158

### CODE REVIEW

**Review by samsondav — CHANGES_REQUESTED** [review]
rename tables also?

**Review by samsondav — CHANGES_REQUESTED** [review]
rename tables also?

**Comment by aalu1418** [comment]
> rename tables also?

I think renaming the tables might be too much for this PR - and also there are ongoing conversations about the persistence layer so I think it would be best to minimize the changes for now

File: common/txmgr/types/tx_store.go [hunk 1]
```diff
@@ -32,52 +32,52 @@ type TxStore[
...
+	LoadTxesAttempts(etxs []*Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], qopts ...pg.QOpt) error
 	MarkAllConfirmedMissingReceipt(chainID CHAIN_ID) (err error)
 	MarkOldTxesMissingReceiptAsErrored(blockNum int64, finalityDepth uint32, chainID CHAIN_ID, qopts ...pg.QOpt) error
-	PreloadEthTxes(attempts []TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], qopts ...pg.QOpt) error
+	PreloadTxes(attempts []TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], qopts ...pg.QOpt) error
 	SaveConfirmedMissingReceiptAttempt(ctx context.Context, timeout time.Duration, attempt *TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], broadcastAt time.Time) error
 	SaveFetchedReceipts(receipts []R, chainID CHAIN_ID) (err error)
 	SaveInProgressAttempt(attempt *TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD]) error
-	SaveInsufficientEthAttempt(timeout time.Duration, attempt *TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], broadcastAt time.Time) error
+	SaveInsufficientAttempt(timeout time.Duration, attempt *TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], broadcastAt time.Time) error
```
Comment (prashantkumar1982) [inline common/txmgr/types/tx_store.go:70]: nit: SaveInsufficientFundsAttempt?
InsufficientAttempt seems a little confusing

File: common/txmgr/types/tx_store.go [hunk 2]
```diff
@@ -32,52 +32,52 @@ type TxStore[
...
 	SetBroadcastBeforeBlockNum(blockNum int64, chainID CHAIN_ID) error
 	UpdateBroadcastAts(now time.Time, etxIDs []int64) error
-	UpdateEthKeyNextNonce(newNextNonce, currentNextNonce SEQ, address ADDR, chainID CHAIN_ID, qopts ...pg.QOpt) error
-	UpdateEthTxAttemptInProgressToBroadcast(etx *Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], attempt TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], NewAttemptState TxAttemptState, incrNextNonceCallback QueryerFunc, qopts ...pg.QOpt) error
-	UpdateEthTxsUnconfirmed(ids []int64) error
-	UpdateEthTxUnstartedToInProgress(etx *Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], attempt *TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], qopts ...pg.QOpt) error
-	UpdateEthTxFatalError(etx *Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], qopts ...pg.QOpt) error
-	UpdateEthTxForRebroadcast(etx Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD], etxAttempt TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE, ADD]) error
+	UpdateKeyNextNonce(newNextNonce, currentNextNonce SEQ, address ADDR, chainID CHAIN_ID, qopts ...pg.QOpt) error
```
Comment (prashantkumar1982) [inline common/txmgr/types/tx_store.go:75]: nit: rename all nonces to sequence

File: core/chains/evm/txmgr/nonce_syncer.go
Comment (prashantkumar1982) [inline core/chains/evm/txmgr/nonce_syncer.go:1]: Maybe another PR, but NonceSyncer should be SequenceSyncer

Comment (aalu1418) [reply core/chains/evm/txmgr/nonce_syncer.go:1]: ah right - good reminder: I will address in a separate ticket: https://smartcontract-it.atlassian.net/browse/BCI-1404

**Review by jmank88 — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]

**Pull request merged**